ext/opentelemetry-ext-flask/tests/base_test.py

```diff
@@ -91,7 +59,7 @@ def assert_environ():
         self.client.get("/assert_environ")
         self.assertEqual(nonstring_keys, set())
 
-    def test_simple(self):
+    def test_simple_uninstrument(self):
```

Comment:
I don't understand the name of this test. `simple_uninstrument` but there are not any calls to uninstrument in the test.

Reply:
Fixed